packages/vue-app/template/client.js

```diff
@@ -488,14 +488,14 @@ function showNextPage(to) {
 // Will not update the instance data, so we have to update $data ourselves
 function fixPrepatch(to, ___) {
   if (this._pathChanged === false && this._queryChanged === false) return
+  
+  const matches = []
+  const instances = getMatchedComponentsInstances(to, matches)
```

Comment:
Why moving this outside of `Vue.nextTick`?

Reply:
Unlike transition out-in mode, in in-out mode,
the "transition In" component instance has already been created in this nextTick,
so "transition In" component instance is included in array returned from the getMatchedComponentsInstances function.

Assuming the purpose of this function,
it is assumed that the newly created component should not be listed in this array.
So to get the same result as out-in mode, the code should be outside the nextTick.

Reply:
Alright sounds good 😄